apps/web-giddh/src/app/purchase/create-purchase-order/create-purchase-order.component.html

```diff
@@ -1080,9 +1306,15 @@
                                         to Select
                                         Account</label>
                                     <p>{{transaction.accountName}}</p>
-                                    <span
-                                        *ngIf="transaction.stockDetails?.uniqueName">({{transaction.stockDetails?.name}}
-                                        )</span>
+                                    <!-- <span
+                                        *ngIf="transaction.stockDetails?.uniqueName">({{transaction.stockDetails?.name}})</span> -->
+
+                                        <span class="d-flex justify-content-end">
+                                            <amount-field [amount]="transaction.stockDetails?.name"
```

Comment:
Why are we showing amount-field component here for stock name?

Reply:
My mistake, corrected